.circleci/config.yml

```diff
@@ -52,7 +19,7 @@ workflows:
           username_envar: "QUAY_USERNAME"
           password_envar: "QUAY_PASSWORD"
           requires:
```

Comment:
While you're here can you add `context: "architect"` to all `push-*` jobs? See https://github.com/giantswarm/architect-orb/pull/149/files

Reply:
Please provide a PR so that I can learn where to put it next time. 🙈